v2client/algod/v2wrappers.go

```diff
@@ -0,0 +1,118 @@
+package algod
+
+import (
+	"context"
+	"fmt"
+	"github.com/algorand/go-algorand-sdk/types"
+	"github.com/algorand/go-algorand-sdk/v2client/common"
+	"github.com/algorand/go-algorand-sdk/v2client/common/models"
+)
+
+// TODO ejr handling of token
+// TODO ejr received peer feedback to have Block return types.Block not models.Block?
+func (client Client) Shutdown(ctx context.Context, timeout models.ShutdownParams, headers ...*common.Header) error {
+	// TODO EJR how to handle private security token
+	return client.post(ctx, nil, "/shutdown", timeout, headers)
+}
+
+func (client Client) RegisterParticipationKeys(ctx context.Context, account string, params models.RegisterParticipationKeysAccountIdParams, headers ...*common.Header) error {
+	// TODO EJR how to handle private security token
+	return client.post(ctx, nil, fmt.Sprintf("/register-participation-keys/%s", account), nil, headers)
+}
```

Comment:
@winder These are marked Private in the specification. Do they need any extra arguments or anything as a result, or can they be handled just like any other endpoint?